impl/src/main/java/org/jboss/cdi/tck/tests/definition/stereotype/broken/alternativeReserve/AlternativeAndReserveOnOneStereotypeAndBeanTest.java

```diff
@@ -34,7 +35,8 @@ public static WebArchive deploy() {
     }
 
     @Test
-    @SpecAssertion(section = DECLARING_RESERVE, id = "c")
+    @SpecAssertions({ @SpecAssertion(section = DECLARING_RESERVE, id = "b"),
+            @SpecAssertion(section = DECLARING_RESERVE, id = "c") })
```

Comment:
`@SpecAssertion` is `@Repeatable`, so you don't need to use `@SpecAssertions` at all.